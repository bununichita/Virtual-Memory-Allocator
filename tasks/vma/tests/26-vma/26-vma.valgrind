==36930== Memcheck, a memory error detector
==36930== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==36930== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==36930== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==36930== Parent PID: 36929
==36930== 
==36930== 
==36930== HEAP SUMMARY:
==36930==     in use at exit: 0 bytes in 0 blocks
==36930==   total heap usage: 40 allocs, 40 frees, 9,062 bytes allocated
==36930== 
==36930== All heap blocks were freed -- no leaks are possible
==36930== 
==36930== For lists of detected and suppressed errors, rerun with: -s
==36930== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
