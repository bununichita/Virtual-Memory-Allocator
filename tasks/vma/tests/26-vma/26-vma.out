Total memory: 3E8 bytes
Free memory: 740 bytes
Number of allocated blocks: 6
Number of allocated miniblocks: 6

Block 1 begin
Zone: 64 - 6E
Miniblock 1:	64	-	6E	| RW-
Block 1 end

Block 2 begin
Zone: 12C - 136
Miniblock 1:	12C	-	136	| RW-
Block 2 end

Block 3 begin
Zone: 190 - 19A
Miniblock 1:	190	-	19A	| RW-
Block 3 end

Block 4 begin
Zone: 1F4 - 1FE
Miniblock 1:	1F4	-	1FE	| RW-
Block 4 end

Block 5 begin
Zone: C8 - DC
Miniblock 1:	C8	-	DC	| RW-
Block 5 end

Block 6 begin
Zone: 2BC - 384
Miniblock 1:	2BC	-	384	| RW-
Block 6 end
1
100
1
300
1
400
1
500
1
200
1
700
